autogen/oai/completion.py

```diff
@@ -187,11 +189,14 @@ def _get_response(cls, config: Dict, raise_on_ratelimit_or_timeout=False, use_ca
         openai.api_key_path = config.pop("api_key_path", openai.api_key_path)
         key = get_key(config)
         if use_cache:
-            response = cls._cache.get(key, None)
-            if response is not None and (response != -1 or not raise_on_ratelimit_or_timeout):
-                # print("using cached response")
-                cls._book_keeping(config, response)
-                return response
+            try:
```

Comment:
try/ excepting checking cache. If checking cache raises an exception it should still make the LLM API call

Reply:
You need to catch the actual exception that occurs under that condition. Otherwise we could be missing a different exception in the same block, and besides the build is failing because of it.

Reply:
`.get(key, None)` does not raise an exception if there is no cache. It just returns `None`. You can remove try/except because it has no effect here.